script.js

```diff
@@ -0,0 +1,115 @@
+const cpf = document.getElementById('cpf')
+const nome = document.getElementById('nome')
+const altura = document.getElementById('altura')
+const idade = document.getElementById('idade')
+let errorMessages = []
+
+
+addEventListener("submit", (e) => {
+    e.preventDefault()
+    isValidForm() ? createDataList() : showErrorMessages()
+    cleanInputs()
+    errorMessages = []
+
+    
+})
+
+
+const validateCpf = () => {
+    return cpf.value.length == 11
+}
+
+const validateNome = () => {
+    return nome.value.length > 0
+}
```

Comment:
É importante verificar se o nome possui apenas letras com ou sem acentuação. Verificar também se há espaços em branco no início e/ou no final do nome (há um método/função JS que faz isso).

Obs.: remover a variável global.